pkg/logger/logger.go

```diff
@@ -488,6 +482,20 @@ func Init(cfg *LoggerConfig) {
 	}
 
 	SetBase(NewLogger(cfg))
+
+	// Flush aggreated logs every 3 seconds
+	if pkgLogger.cfg.Aggregate {
+		go func() {
+			for range time.Tick(3 * time.Second) {
+				for lo, count := range pkgLogger.LogCount.Flush() {
+					Log(lo.Level, false, lo.Msg,
```

Comment:
`innerAggregation bool` (Log argument) is always called as false. Is the argument really needed now?

Reply:
What is the difference between inner and "outer" aggregation?

Reply:
> What is the difference between inner and "outer" aggregation?

In case we got bpf logs from perfbuf, they're already aggregated, so pkg logger would consider them as outer aggregates and won't count them internally.

Same here for the use of an external logger trigger (libbpfgo), we can't track the origin of such logs, so inner aggregation must be turned off.

> innerAggregation bool (Log argument) is always called as false. Is the argument really needed now?

Log is a generic log helper. We don't have any use of it with innerAggregation as true indeed. Perhaps to remove that parameter can be part of https://github.com/aquasecurity/tracee/issues/2475